params/config.go

```diff
@@ -185,6 +187,8 @@ var (
 		ShanghaiTime:                  newUint64(0),
 		CancunTime:                    newUint64(0),
 		Enable4844:                    false,
+		// TODO(0xHansLee): Need to bake timestamp for Odyssey hard fork
+		EIP1559Denominator: newUint64(24),
```

Comment:
Do we need this parameter in the chainConfig? This parameter setting should be 24 only after upgrade height. Other developers may be confused to think that it is the default setting for odyssey network.

Reply:
To prevent people being confused, I can add postfix with the name of hardfork. 
The reason why I added this to chainConfig is to make this configurable.

Reply:
Who should be configuring it and from where? I thought it is always tied to upgrade height

Reply:
Yes it is always tied to the upgrade height. 
I meant it can be dependent on the network. For example, we set the denom by 24 for Odyssey at this time, but we can set different denom for mainnet (e.g. 48) based on our testnet experience.

Reply:
@0xHansLee Then I'm good with this configuration if this is to configure for different network.

Reply:
Right!